src/db/migrations/20250218094907_alter_and_add_profile_details_to_onerep_profiles_table.js

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export function up(knex) {
+  return knex.schema.alterTable("onerep_profiles", async (table) => {
+    table
+      .specificType("first_names", "character varying(255)[]")
```

Comment:
With backend/DB design not being my speciality, I don't want to have a strong opinion here, but I suppose the following are reasons why it's OK to use array types and JSON rather than the more cumbersome foreign tables:

- We don't care about DBMS portability,
- we won't be selecting rows based on the contents of these columns,
- we'll rarely write to these columns, and
- when we do write to them, we'll often update all elements in one go.

Writing this down more for myself, but happy to learn if anyone has corrections or additions.

Reply:
I’ve considered adding a new table, but that might be too much overhead. We are already using `jsonb` in a couple of places. That means:
- we don’t have DBMS portability either way
- I don’t anticipate that we need to select rows based on these columns
- writing to those columns happens only if users update their profile info
- we only update the elements in those columns all at once